src/platform/forms-system/src/js/fields/ArrayField.jsx

```diff
@@ -10,10 +10,11 @@ import {
   deepEquals,
 } from '@department-of-veterans-affairs/react-jsonschema-form/lib/utils';
 
+import scrollTo from 'platform/utilities/ui/scrollTo';
 import { scrollToFirstError } from '../utilities/ui';
 import { setArrayRecordTouched } from '../helpers';
 import { errorSchemaIsValid } from '../validation';
-import { isReactComponent } from '../../../../utilities/ui';
+import { getScrollOptions, isReactComponent } from '../../../../utilities/ui';
 
 const Element = Scroll.Element;
 const scroller = Scroll.scroller;
```

Comment:
Do we still need these?

Reply:
the Element is still being referenced further down, however I can remove the scroller reference. Will push that up shortly.